intersectionLinkedLists.js

```diff
@@ -0,0 +1,42 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/intersection-of-two-linked-lists/
+
+const getIntersectionNode = function (headA, headB) {
+  let depthA = 0;
+  let depthB = 0;
+  let h1 = headA;
+  let h2 = headB;
+  while (h1) {
+    depthA += 1;
+    h1 = h1.next;
+  }
+  while (h2) {
+    depthB += 1;
+    h2 = h2.next;
+  }
```

Comment:
Lets create a helper function to find list depth and re-use it here (instead of duplicating `while` loop twice).